tests/php/Integration/Run/Command/Repl/ReplCommandTest.php

```diff
@@ -167,4 +159,46 @@ private function createReplTestIo(): ReplTestIo
 
         return new ReplTestIo($exceptionPrinter);
     }
+
+    private function prepareRunDependencyProvider(ReplCommandIoInterface $io): void
+    {
+        AbstractClassResolver::resetCache();
+
+        AnonymousGlobal::overrideExistingResolvedClass(
+            '\module-name@anonymous\ReplCommandTest\Config',
+            new RunConfig()
+        );
+
+        AnonymousGlobal::overrideExistingResolvedClass(
+            '\module-name@anonymous\ReplCommandTest\DependencyProvider',
+            new RunDependencyProvider()
+        );
```

Comment:
We can also remove these lines thanks to this PR: https://github.com/gacela-project/gacela/pull/155/files